themes/default/content/docs/intro/console/projects-and-stacks.md

```diff
@@ -29,11 +27,31 @@ To create a project:
 1. Select **Create project**.
 1. Follow the provided CLI command instructions.
 
-## Managing your Stacks
+## Stack Permissions
+
+Stack permissions are based on the member's role within the organization, and on their team membership.
+Additionally, any member who creates a stack will be granted admin permissions on that stack.
+
+Organization admins can control the stack default permissions at the organization level from the organization's **Settings**.
+There are four types of stack permissions: `None`, `Read`, `Write`, and `Admin`.
+[Team permissions]({{< relref "/docs/intro/console/teams#team-permissions" >}}) will expand these default permissions.
 
-Every Pulumi program is deployed to a [stack]({{< relref "/docs/intro/concepts/stack" >}}). If you have followed the instructions after creating your project, you can view your newly-created stack in the Pulumi Console unless you have [explicitly opted out]({{< relref "/docs/intro/concepts/state" >}}). The Pulumi Console provides safe locking so that your resource state can never get corrupted by a concurrent update.
+Stack permissions allow users to perform the following actions:
 
-### Viewing your organization stacks
+| Action | None | Read | Write | Admin |
+|--------|------|------|-------|-------|
+| View update history | | ✅ | ✅ | ✅ |
+| Decrypt secret configuration | | ✅ | ✅ | ✅ |
+| Read stack resources | | ✅ | ✅ | ✅ |
+| Preview stack changes | | ✅ | ✅ | ✅ |
+| Update stack | | | ✅ | ✅ |
+| Destroy stack (`pulumi destroy`) | |   | ✅ | ✅ |
+| Export stack checkpoint | | ✅ | ✅ | ✅ |
+| Import stack checkpoint |  | | ✅ | ✅ |
+| Delete stack (`pulumi stack rm`) | | | | ✅ |
+| Transfer to another organization | | | | ✅ |
+
```

Comment:
We seem to have lost the Assigning Stack Permissions section: https://www.pulumi.com/docs/intro/console/stack-permissions/#assigning-stack-permissions

Any reason not to bring that forward?

Reply:
nope, that was a mistake

Reply:
oh sorry i misread this, i thought you were commenting on the table.

Reply:
this content is now in the stack permissions section, and on the team page